src/it/scala/uk/gov/hmrc/hmrcfrontend/views/example/hmrcAddToAListIntegrationSpec.scala

```diff
@@ -14,9 +14,7 @@ class hmrcAddToAListIntegrationSpec extends TemplateIntegrationSpec {
   val request: RequestHeader = FakeRequest()
   val messages: Messages     = MessagesSupport().messages
```

Comment:
Can these values `request` and `messages` be deleted as those tests are no longer run?

Reply:
Re-instated applicable tests.